app/controllers/navigation/Navigator.scala

```diff
@@ -182,21 +195,52 @@ object Navigator {
     case page                  => throw new IllegalArgumentException(s"Navigator back-link route not implemented for $page on occasional")
   }
 
+  val universal: PartialFunction[DeclarationPage, Mode => Call] = {
```

Comment:
Can we use the better name for it? If I'm understanding this correctly what do you think about `common`?

Reply:
Would be fixed in part2